services/private_ip_manager/src/main/java/com/futurewei/alcor/privateipmanager/controller/IpAddrController.java

```diff
@@ -82,6 +85,7 @@ private int checkIpVersion(String ipAddress) throws Exception {
     @PostMapping("/ips")
     @ResponseBody
     @ResponseStatus(HttpStatus.CREATED)
+    @DurationStatistics
```

Comment:
This is a very good annotation. Consider to apply it to all other services as well 👍